library/FrontRow/App/Database.hs

```diff
@@ -42,9 +44,11 @@ type SqlPool = Pool SqlBackend
 
 class HasSqlPool app where
   getSqlPool :: app -> SqlPool
+  getStatementTimeoutSeconds :: app -> Maybe Int
```

Comment:
Is this worth a `newtype`?

```suggestion
  getStatementTimeout :: app -> Maybe Seconds
```

Reply:
I also wonder if we want to actually put a (somewhat high) default here. Having a default was more about not requiring code changes in all users right away; I'm not sure we want "unlimited" to be that default.

Reply:
Love it. Done and looking good